vault/data_source_kv_secret_v2_test.go

```diff
@@ -29,6 +29,10 @@ func TestDataSourceKVV2Secret(t *testing.T) {
 					resource.TestCheckResourceAttr(resourceName, consts.FieldName, name),
 					resource.TestCheckResourceAttr(resourceName, consts.FieldPath, fmt.Sprintf("%s/data/%s", mount, name)),
 					resource.TestCheckResourceAttr(resourceName, "destroyed", "false"),
+					resource.TestCheckResourceAttr(resourceName, "data.zip", "zap"),
```

Comment:
Probably want to check the length of `data.%`

Reply:
Added in latest commit